scripts/update-custodian.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
```

Comment:
Can we add -u and -o pipefail to this? http://blog.kablamo.org/2015/11/08/bash-tricks-eux/